src/custom/utils/signatures.ts

```diff
@@ -137,16 +137,19 @@ async function _signPayload(
   payload: any,
   signFn: typeof _signOrder | typeof _signOrderCancellation,
   signer: Signer,
-  signingMethod: 'v4' | 'int_v4' | 'v3' | 'eth_sign' = 'v4'
+  signingMethod: 'default' | 'v4' | 'int_v4' | 'v3' | 'eth_sign' = 'v4'
 ): Promise<SigningResult> {
   const signingScheme = signingMethod === 'eth_sign' ? SigningScheme.ETHSIGN : SigningScheme.EIP712
   let signature: Signature | null = null
 
   let _signer
   try {
     switch (signingMethod) {
+      case 'default':
+        _signer = new TypedDataVersionedSigner(signer)
```

Comment:
If we use the default, we don't specify the version. Will use `eth_signTypedData `